airflow/configuration.py

```diff
@@ -93,6 +93,9 @@ def run_command(command):
         'sql_alchemy_pool_recycle': 3600,
         'dagbag_import_timeout': 30,
     },
+    'operators': {
+        'default_owner': 'Airflow'
```

Comment:
Since this is supposed to be a unix-y thing, perhaps `airflow` instead of `Airflow`?